From 1696a285897803c426b5d7c01cc7fbb2aebe9f3e Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Tue, 29 Mar 2022 13:37:59 +0300
Subject: [PATCH 2/3] ddr: Rename compatible of U-Boot DT '/ddr' node

Renamed the compatible of the U-Boot's DT '/ddr' node to
align with U-Boot and Linux.

Issue: ALB-8586
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32_bl2_el3.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/plat/nxp/s32/s32_bl2_el3.c b/plat/nxp/s32/s32_bl2_el3.c
index 340ed37e2..23675ce1a 100644
--- a/plat/nxp/s32/s32_bl2_el3.c
+++ b/plat/nxp/s32/s32_bl2_el3.c
@@ -376,15 +376,15 @@ static int ft_fixup_ddr_errata(void *blob)
 		return 0;
 	}
 
-	nodeoff = fdt_node_offset_by_compatible(blob, -1, "fsl,s32gen1-ddr");
+	nodeoff = fdt_node_offset_by_compatible(blob, -1, "nxp,s32cc-ddr");
 	if (nodeoff < 0) {
-		ERROR("Failed to get offset of 'fsl,s32gen1-ddr' node\n");
+		ERROR("Failed to get offset of 'nxp,s32cc-ddr' node\n");
 		return nodeoff;
 	}
 
 	ret = fdt_setprop_string(blob, nodeoff, "status", "okay");
 	if (ret) {
-		ERROR("Failed to enable 'fsl,s32gen1-ddr' node\n");
+		ERROR("Failed to enable 'nxp,s32cc-ddr' node\n");
 		return ret;
 	}
 
-- 
2.17.1

